basic/Jenkinsfile

```diff
@@ -2,7 +2,7 @@ pipeline {
   agent {
     // this image provides everything needed to run Cypress
     docker {
-      image 'cypress/base:10'
+      image 'cypress/base:latest'
```

Comment:
Cypress v12 dropped node 12 support. This is an example for users. Updated to pull latest.